### DISCUSSION TITLE
v1.6.0-2

### DISCUSSION THREAD

**Discussion by jufab** [jufab/opentelemetry-angular-interceptor#184] [General]
## What's Changed
* chore: update dev dependencies by @jufab in https://github.com/jufab/opentelemetry-angular-interceptor/pull/183

**Full Changelog**: https://github.com/jufab/opentelemetry-angular-interceptor/compare/v1.6.0-1...v1.6.0-2

<hr /><em>This discussion was created from the release <a href='https://github.com/jufab/opentelemetry-angular-interceptor/releases/tag/v1.6.0-2'>v1.6.0-2</a>.</em>

**Comment by 314gui** [comment]
Getting an error installing @jufab/opentelemetry-angular-interceptor.
@opentelemetry/context-zone-peer-dep": "**1.17.1.**"(one more dot) doesn't exist on NPM and its a dependency on **projects/opentelemetry-interceptor/package.json**

After versions contain a breaking changes.
